R/supervised-model-deployment.R

```diff
@@ -244,13 +246,42 @@ SupervisedModelDeployment <- R6Class("SupervisedModelDeployment",
       print('Raw data set after creating dummy vars (for top 3 factors only)')
       print(str(private$dfTestRaw))
     }
+  },
+  
+  loadModelAndInfo = function(modelFullName) {
+    # Try to load the model
+    tryCatch({
+      # Set file names for model and associated information
+      fitObjFile <- paste("rmodel_probability_", self$params$modelName, ".rda", 
+                          sep = "")
+      modelInfoFile <- paste("rmodel_info_", self$params$modelName, ".rda", 
+                             sep = "")
+      
+      load(modelInfoFile)  # Get model info
+      self$modelInfo <- modelInfo
+      load(fitObjFile) # Produces fit object (for probability)
+      self$fitObj <- fitObj
```

Comment:
I wanted to pass the function loadModelAndInfo an additional parameter to directly assign fitObj to fitRF, fitGrLasso, etc. but couldn't figure out how to pass a reference (rather than a copy) to a function (see below) so I added a fitObj variable to SupervisedModelDeployment and assign this to fitRF, fitGRLasso, etc. in each of the individual classes.

Example code showing how R passes function arguments as copies rather than references:
```
unknown_fruit <- NA

assign_banana = function(fruit) {
  fruit <- "Banana"
}

assign_banana(unknown_fruit) # doesn't change the value of unknown_fruit
is.na(unknown_fruit) # returns TRUE
```